warehouse/ingest-core/src/main/java/datawave/IdentityDataType.java

```diff
@@ -1,15 +1,16 @@
 package datawave;
 
 import java.util.Collection;
+import datawave.data.type.Type;
 
-public class IdentityDataType implements datawave.data.type.Type {
```

Comment:
Original error was in `EventMetadataTest` 
```
Type mismatch: cannot convert from List<IdentityDataType> to List<Type<?>>`
```

Previouly, `IdentityDataType` wasn't parameterized and resulted in a warning.